logger.go

```diff
@@ -22,76 +35,194 @@ const (
 	DebugLevel Level = "debug"
 	// SpamLevel level. Give me EVERYTHING.
 	SpamLevel Level = "spam"
+	// DefaultLevel is an alias for "info" level.
+	DefaultLevel = InfoLevel
 )
 
+var (
+	standardLogger = New(os.Stdout)
+	errorLogger    = New(os.Stderr).WithLevel(ErrorLevel)
+	levelsMap      = map[string]Level{
+		"fatal":   FatalLevel,
+		"error":   ErrorLevel,
+		"warn":    WarnLevel,
+		"info":    InfoLevel,
+		"verbose": VerboseLevel,
+		"debug":   DebugLevel,
+		"spam":    SpamLevel,
+	}
+)
+
+// ParseLevel converts level name to Level. It is case insensitive, returns DefaultLevel if value cannot be converted.
+func ParseLevel(s string) (level Level, ok bool) {
+	level, ok = levelsMap[strings.ToLower(s)]
+	if !ok {
+		level = DefaultLevel
+	}
+	return level, ok
+}
+
 // Logger.
-type Logger struct{}
+type Logger struct {
+	output     io.Writer
+	tags       []string
+	level      Level
+	linePrefix string
+}
 
 // New creates new instance of Logger.
```

Comment:
I would give some examples - what writers may be used

Reply:
I've rewritten examples completely. Readme contains two basic examples now. Rest of the examples is in the logger_test.go file.